charts/karpenter/values.yaml

```diff
@@ -173,29 +156,21 @@ settings:
   # faster than this time, the batching window will be extended up to the maxDuration. If they arrive slower, the pods
   # will be batched separately.
   batchIdleDuration: 1s
-  # -- Azure-specific configuration values
-  azure:
-    # -- Cluster name.
-    clusterName: ""
-    # -- Cluster endpoint.
-    clusterEndpoint: ""
-    # -- Kubelet client TLS bootstrap token.
-    kubeletClientTLSBootstrapToken: ""
-    # -- SSH public key.
-    sshPublicKey: ""
-    # -- Network plugin.
-    networkPlugin: "azure"
-    # -- Network policy.
-    networkPolicy: ""
-    # -- The VM memory overhead as a percent that will be subtracted from the total memory for all instance types
-    vmMemoryOverheadPercent: 0.075
-    # -- The global tags to use on all Azure infrastructure resources (VMs, etc.)
-    # TODO: not propagated yet ...
-    tags:
+  # -- Cluster CA bundle for TLS configuration of provisioned nodes. If not set, this is taken from the controller's TLS configuration for the API server.
+  clusterCABundle: ""
+  # -- Cluster name.
+  clusterName: ""
+  # -- Cluster endpoint.
+  clusterEndpoint: ""
+  # -- The VM memory overhead as a percent that will be subtracted from the total memory for all instance types
+  vmMemoryOverheadPercent: 0.075
+  # -- The global tags to use on all Azure infrastructure resources (VMs, etc.)
+  # TODO: not propagated yet ...
+  tags:
+
   # -- Feature Gate configuration values. Feature Gates will follow the same graduation process and requirements as feature gates
   # in Kubernetes. More information here https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features
   featureGates:
-    # -- drift is in ALPHA and is disabled by default.
-    # Setting drift to true enables the drift disruption method to watch for drift between currently deployed nodes
-    # and the desired state of nodes set in provisioners and node templates
-    drift: true
```

Comment:
It was promoted to stable for v1 right?

Reply:
Yes, and that's why it is not behind a feature gate anymore and why this block is removed